src/server/service/match.ts

```diff
@@ -318,13 +349,28 @@ export class Match {
         player.isScoreSaved = true;
 
         if (player.isBot) return;
-
-        await dbTx
+        await tx
           .update(users)
           .set({
             score: sql`${users.score} + ${player.score}`,
+            botsBusted: sql`${users.botsBusted} + ${player.botsBusted}`,
           })
           .where(eq(users.id, player.userId));
+
+        const playerAchievements = player.achievements
+          .filter((achievement) => {
```

Comment:
for this arrow functions that immediately return an object or boolean you can declare the like:
```TS
(yourInput) => (yourReturnObjectOrBoolean)
```

No need to open up brackets